<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zamowienie_pracownik_info</class>
 <widget class="QDialog" name="Zamowienie_pracownik_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>379</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>379</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>348</width>
     <height>336</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Nr ID zamówienia:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Produkt:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="produkt">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Cena produktu:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="cena">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Liczba sztuk:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Przesyłka:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="przesylka">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>Cena przesyłki:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="cena_przesylki">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>Łącznie do zapłaty</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLabel" name="do_zaplaty">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Data zamówienia:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="data_zamowienia">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLabel" name="status">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="id">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Opis produktu: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>Kategoria:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="kategoria">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="opis">
        <property name="minimumSize">
         <size>
          <width>219</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>219</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="sztuki">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>Użytkownik:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="login">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="powrot">
      <property name="text">
       <string>Powrót</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
